src/pages/rekey-search/rekey-search.html

```diff
@@ -64,7 +64,7 @@ <h3>
       </tab>
     </tabs>
 
-    <div class='results-error' *ngIf="(pageState.hasSearched$ | async) && searchResults.length === 0">
+    <div class='results-error' *ngIf="(pageState.hasSearched$ | async) && isBookedTestSlotEmpty(pageState.bookedTestSlot | async)">
```

Comment:
are we missing a `$` on the end of `pageState.bookedTestSlot` here?

Reply:
Not anymore heh